fuel-vm/src/interpreter/blockchain.rs

```diff
@@ -96,6 +96,7 @@ where
     /// contract_code = contracts[contract_id]
     /// mem[$ssp, $rC] = contract_code[$rB, $rC]
     /// ```
+    /// Returns size of contract
```

Comment:
Is this comment accurate? The return type appears to be `IoResult<(), S::DataError>`.

Reply:
Thanks. Left over from earlier iteration.